File: points_of_interest/v1/by_id/ios/PointsOfInterest.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

amadeus.referenceData.locations.pointOfInterest(poiId: "8DA7B6CDCA").get(
    onCompletion: {
        response, error in
        if error == nil {
            print(response!.data)
        }
    })

File: points_of_interest/v1/by_id/Node SDK/points_of_interest.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// Extract the information about point of interest with ID '9CB40CB5D0'
amadeus.referenceData.locations.pointOfInterest('9CB40CB5D0').get()
  .then(function (response) {
    console.log(response);
  }).catch(function (response) {
    console.error(response);
  });